pkg/koordlet/runtimehooks/hooks/cpuset/cpuset.go

```diff
@@ -93,6 +98,48 @@ func (p *cpusetPlugin) SetContainerCPUSet(proto protocol.HooksProtocol) error {
 	return nil
 }
 
+func (p *cpusetPlugin) SetPodCPUQuota(proto protocol.HooksProtocol) error {
+	podCtx := proto.(*protocol.PodContext)
+	if podCtx == nil {
+		return fmt.Errorf("pod protocol is nil for plugin %v", name)
+	}
+	req := podCtx.Request
+
+	// cpuset from pod annotation (LSE, LSR)
+	// NOTE: unset cfs quota for cpuset pods to avoid unexpected throttles.
+	// https://github.com/koordinator-sh/koordinator/issues/489
+	if cpusetVal, err := getCPUSetFromPod(req.Annotations); err != nil {
```

Comment:
define an independent checking function in a new file for compatibility adaption.

Reply:
done